packages/flutter_tools/lib/src/commands/config.dart

```diff
@@ -115,8 +115,8 @@ class ConfigCommand extends FlutterCommand {
 
     if (argResults.wasParsed('analytics')) {
       final bool value = boolArg('analytics');
-      flutterUsage.enabled = value;
       AnalyticsConfigEvent(enabled: value).send();
+      flutterUsage.enabled = value;
```

Comment:
Maybe add a comment here for future readers?

Reply:
Done.